.gitignore

```diff
@@ -12,6 +12,7 @@
 /Upgrading.txt
 /flow
 /flow.bat
+/.idea/
```

Comment:
this should be added to some global .gitignore since it's not immediately related to this project/repository (IMO)

Reply:
Does no harm, though.